app/components/blacklight/gallery/slideshow_component.rb

```diff
@@ -21,10 +21,9 @@ def slideshow_tag(image_options = { alt: '' })
           @view_context.send(method_name, @document, image_options)
         elsif view_config.slideshow_field
           url = slideshow_image_url
-
           image_tag url, image_options if url.present?
         elsif presenter.thumbnail.exists?
-          presenter.thumbnail.thumbnail_tag(image_options, url_options.reverse_merge(suppress_link: true))
+          presenter.thumbnail.thumbnail_tag(image_options, url_options)
```

Comment:
Should we also link images that come from the slideshow_field (maybe there's an argument for leaving slideshow_method as-is and letting the downstream consumer deal with it)?